src/app/public/modules/numeric/numeric.service.ts

```diff
@@ -152,6 +142,60 @@ export class SkyNumericService {
     return output;
   }
 
+  /**
+   * Rounds a given number
+   *
+   * JS's limitation - numbers bigger than Number.MIN_SAFE_INTEGER or Number.MAX_SAFE_INTEGER
+   * are not guaranteed to be represented or rounded correctly
+   * @param value - value to round
+   * @param precision - what precision to round with, defaults to 0 decimal places
+   */
+  private roundNumber(
+    value: number,
+    precision: number = 0
+  ): number {
+    if (precision < 0) { throw new Error('SkyInvalidArgument: precision must be >= 0'); }
+​
+    /* tslint:disable-next-line:no-null-keyword */
+    /* Sanity check - ignoring coverage but should not ignore if we make this method public */
+    /* istanbul ignore next */
+    if (isNaN(value) || value === null) { return 0; }
```

Comment:
`return` should be on a new line.

Reply:
I'm not sure I understand why this is ignored only if it's public? If it's a case that will never occur as it stands today, why include the condition?